upload-metadata.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "http://qbic.life/v1/upload-metadata.schema.json",
+    "title": "Upload metadata for data registration at QBiC",
+    "description": "A full description of mandatory and optional metadata properties that need to/can be included for data registration via QBiC dropboxes.",
+    "type": "object",
+    "definitions": {
+        "qc": {
+            "type": "object",
+            "properties": {
+                "qcml_id": {
+                    "type": "string",
+                    "pattern": "^QC:[0-9]{7}$"
+                }, 
+                "name": {
+                    "type": "string",
+                    "description": "Name of the quality control",
+                    "examples": ["read count", "target region read depth", "Q20 read percentage"]
+                },
+                "value": {
+                    "type": "string"
+                }
+            }
+        },
+        "sample": {
+            "type": "object",
+            "properties": {
+                "genome": {
+                    "type": "string",
+                    "examples": ["GRCh37"]
+                },
+                "id_genetics": {
+                    "type": "string",
+                    "examples": ["GS000000_01"]
+                },
```

Comment:
maybe add a `"description" : ` here. I do not understand what `id_genetics` contains. What sort of id? Where does it come from? Can it be multiple ids from different sources?